physutils/tests/test_tasks.py

```diff
@@ -0,0 +1,107 @@
+"""Tests for physutils.tasks and their integration."""
+
+import os
+
+import physutils.tasks as tasks
+from physutils import physio
+from physutils.tests.utils import create_random_bids_structure
+
+
+def test_generate_physio_phys_file():
+    """Test generate_physio task."""
+    physio_file = os.path.abspath("physutils/tests/data/ECG.phys")
+    task = tasks.generate_physio(input_file=physio_file, mode="physio")
+    assert task.inputs.input_file == physio_file
+    assert task.inputs.mode == "physio"
+    assert task.inputs.fs is None
+
+    task()
+
+    physio_obj = task.result().output.out
+    assert isinstance(physio_obj, physio.Physio)
+    assert physio_obj.fs == 1000
+    assert physio_obj.data.shape == (44611,)
+
+
+def test_generate_physio_bids_file():
+    """Test generate_physio task."""
+    create_random_bids_structure("physutils/tests/data", recording_id="cardiac")
+    bids_parameters = {
+        "subject": "01",
+        "session": "01",
+        "task": "rest",
+        "run": "01",
+        "recording": "cardiac",
+    }
+    bids_dir = os.path.abspath("physutils/tests/data/bids-dir")
+    task = tasks.generate_physio(
+        input_file=bids_dir,
+        mode="bids",
+        bids_parameters=bids_parameters,
+        bids_channel="cardiac",
```

Comment:
This is failing since bids_channel is not a valid argument in the `generate_physio` function

Reply:
Ok just realized looking at the previous comments that `bids_channel` was changed to` col_physio_type` in generate_physio, but the `test_tasks.py` have not been update accordingly